handler/date/date.go

```diff
@@ -21,14 +22,35 @@ func Memory(c *gin.Context){
 	month:=Date.Month
 	day:=Date.Day
 	claim,err:=token.ParseToken(user.User.Token)

```

Comment:
IMO, maybe use a gin middleware to parse auth information, like token, is better.